src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -881,6 +883,32 @@ public void testListTransfer() throws Exception {
         assertThat(returnValue.getData().get(0).getForeignExchanges().get(0).getRate(), is(equalTo(0.79)));
     }
 
+    //
+    //TransferMethodConfiguration
+    //
+
+    @Test
+    public void testGetTransferMethodConfiguration() throws Exception {
+        String functionality = "getTransferMethodConfiguration";
+        initMockServer(functionality);
+
+        HyperwalletTransferMethodConfiguration returnValue;
+        try {
+            returnValue = client.getTransferMethodConfiguration("usr-b86ab524-2787-46e2-b536-5a70e37349f7","US","USD", Type.BANK_ACCOUNT, ProfileType.INDIVIDUAL);
+        } catch (Exception e) {
+            mockServer.verify(parseRequest(functionality));
+            throw e;
+        }
+
+        assertThat(returnValue.getCountries().get(0), is(equalTo("US")));
+        assertThat(returnValue.getCurrencies().get(0), is(equalTo("USD")));
+        assertThat(returnValue.getFields().get(0).getDataType(), is(equalTo(DataType.NUMBER)));
```

Comment:
Pls test the fields with the new enum values added in the PR(DATE, PHONE).

Reply:
It was tested for the countries with issues - Pakistan and Brazil. Issue could be reproduced and it was resolved with the change. Phone and date are not search fields. Rather they cause problem for these two countries as they seem to have been added after introduction of V4. These fields were not identified by V3.

Reply:
Functional tests were done for user's version, support version and this version.

Reply:
Functional tests were done for user's version, support version and this version.